demo/hide-header.html

```diff
@@ -22,9 +22,13 @@
   <link rel="import" href="../../paper-toolbar/paper-toolbar.html">
   <link rel="import" href="../../iron-icons/iron-icons.html">
   <link rel="import" href="../../paper-icon-button/paper-icon-button.html">
-  <link rel="stylesheet" href="../../paper-styles/demo.css">
+
+  <link rel="import" href="../../paper-styles/color.html">
+  <link rel="import" href="../../iron-flex-layout/iron-flex-layout.html">
   <link rel="import" href="sample-content.html">
 
+  <link rel="stylesheet" href="../../paper-styles/demo.css">
```

Comment:
While you're at it, you should move this to Polymer 1.1.0, and use a style import